### PR TITLE
Return Trip date picker selection updates 

### PR DESCRIPTION
**Pull request by julesian**
allow Return Trip date picker to pre-select out of range dates

### CODE REVIEW

File: Sources/TripKitUI/views/results/TKUITimePickerSheet.swift [hunk 1]
```diff
@@ -309,15 +320,43 @@ public class TKUITimePickerSheet: TKUISheet {
...
+    
+    var title = Loc.Done
+    var isEnabled = true
+    
+    if let earliestDate = config.minimumDate, selectedDate < earliestDate {
+      title = "Below earliest date"
+      isEnabled = false
+    } else if let latestDate = config.maximumDate, selectedDate > latestDate {
+      title = "Beyond furthest date"
+    }
```
Comment (nighthawk) [inline Sources/TripKitUI/views/results/TKUITimePickerSheet.swift:354]: These should be localisable. Please add appropriate entries to `Loc.swift`.

File: Sources/TripKitUI/views/results/TKUITimePickerSheet.swift [hunk 2]
```diff
@@ -309,15 +320,43 @@ public class TKUITimePickerSheet: TKUISheet {
 
   @objc
   func doneButtonPressed(sender: Any) {
+    guard doneSelector.isEnabled 
+    else {
+      // TODO: Alert?
```
Comment (nighthawk) [inline Sources/TripKitUI/views/results/TKUITimePickerSheet.swift:325]: Can you clarify this TODO? Is it safe to ignore, or when would this happen?

**Review by nighthawk — CHANGES_REQUESTED** [review]

File: Sources/TripKitUI/views/results/TKUITimePickerSheet.swift (hunk 2 above)
Comment (julesian) [reply Sources/TripKitUI/views/results/TKUITimePickerSheet.swift]: this was a left over of the previous implementation that I made, updated.

**Review by nighthawk — APPROVED** [review]
LGTM

**Pull request merged**